app/pandas.ipynb

```diff
@@ -93,18 +94,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "print(df)",
```

Comment:
For Task 3.1, you need to display only the 'Age' column. Use `print(df['Age'])` instead of `print(df)` to fulfill the task requirement.